source/data/tier1data/schemas/metadata-schemas.rst

```diff
@@ -0,0 +1,452 @@
+.. _metadata-schemas:
+
+####################################
+Metadata schemas in the ManGO portal
+####################################
+
+
+
+This article describes the ManGO portal functionalities related to
+metadata schemas: how to design them and how to apply them. Users who
+might want to design their own schemas independently and load them via
+JSON, as well as developers interested in implemented this feature
+outside the portal, are directed to `the technical
+specifications <metadata-schemas-tech.qmd>`__.
+
+One crucial principle of the metadata schema functionality in the ManGO
+portal is that schemas that can be used to apply metadata cannot be
+modified. In other words, for a schema to be used in metadata annotation
+it must be fixed and stable; a schema that is undergoing changes cannot
+reliably be used for annotation. However, during the course of a
+research project it might be necessary to update a schema, and it would
+be impractical to create whole new schemas with different names for
+every such change. In order to tackle this issue, the ManGO portal
+implements a lifecycle via versioning and tags. In short, a schema can
+evolve and its evolutions can be registered as new versions of the same
+schema. Each version can have one of three statuses: (1) ‘draft’, while
+it is being designed and edited; (2) ‘published’, when it is ready to be
+implemented; and (3) ‘archived’, when it should not be used anymore,
+maybe because a new version has been published.
+
+In this context, the rest of this document will walk you through the
+process of creating, managing and applying metadata schemas. First,
+`Section 1 <#sec-draft>`__ will illustrate how to design a new schema
+from scratch and save a draft. Then, `Section 2 <#sec-lifecycle>`__ will
+briefly discuss the stages of a schema in more detail, including how
+they can be managed in the Metadata Schema Manager (the “Metadata
+schemas” tab in the ManGO portal). Finally,
+`Section 3 <#sec-application>`__ will show how a published schema can be
+used to annotate metadata.
+
+.. _sec-draft:
+
+1. Schema design
+================
+
+In order to design a new schema, we need to go to the “Metadata schemas”
+section via the left sidebar menu. This will first show a selection of
+projects (“realms”) to which your schemas may belong. After choosing
+one, the schema manager per se is shown (see `Figure 1 <#fig-start>`__):
```

Comment:
Once `numfig` is enabled in the configuration file, this can be replace by
```suggestion
one, the schema manager per se is shown (see :numref:`fig-start-a` and :numref:`fig-schemaeditor`):
```
and all other references to figures accordingly.

Reply:
I'll apply it, but note that it becomes impossible to reference a group of pictures (a grid), which is something the author of this page used a lot.
I'll just turn these reference into plaintext (e.g. 'See figure 1'), but if you know a way to do it with a link, that's always welcome :)